testrunner-gradle-plugin/src/main/resources/META-INF/gradle-plugins/testrunner.properties

```diff
@@ -0,0 +1 @@
+implementation-class=edu.illinois.cs.testrunner.gradleplugin.TestPluginPlugin
```

Comment:
Yes, we do need this. Similar to nondex-gradle-plugin: https://github.com/TestingResearchIllinois/NonDex/blob/master/nondex-gradle-plugin/src/main/resources/META-INF/gradle-plugins/edu.illinois.nondex.properties, we need to explicitly specify the Plugin class in gradle so that the gradle knows we are building a gradle plugin instead of a normal project.